checkov/terraform/checks/resource/azure/SQLServerNoPublicAccess.py

```diff
@@ -15,7 +15,8 @@ def __init__(self):
         super().__init__(name=name, id=id, categories=categories, supported_resources=supported_resources)
 
     def scan_resource_conf(self, conf):
-        if 'start_ip_address' in conf and conf['start_ip_address'][0] in ['0.0.0.0', '0.0.0.0/0']: # nosec
+        if 'start_ip_address' in conf and conf['start_ip_address'][0] in ['0.0.0.0', '0.0.0.0/0'] and \
+            'end_ip_address' in conf and conf['end_ip_address'][0] == '255.255.255.255': # nosec
```

Comment:
there is limitation on how `bandit` scans multiline statements, you need to add the `# nosec` on the first line of code and I can also recommend to add `()` brackets arround the whole statement, then you don't need the backslash.
```python
    def scan_resource_conf(self, conf):
        if ('start_ip_address' in conf and conf['start_ip_address'][0] in ['0.0.0.0', '0.0.0.0/0'] and  # nosec
                'end_ip_address' in conf and conf['end_ip_address'][0] == '255.255.255.255'):
            return CheckResult.FAILED
        return CheckResult.PASSED
```

Reply:
Thanks, I've used your suggested code directly in b38e93e!